docs-13092-acm-experience-experience-builder-page-not-default-jh

JennyHajee opened the pull request:
Updated the Diagnostics topic with steps to configure Login & Registration defaults  back to Experience Builder Page.

JennyHajee committed:
docs-13092-acm-experience-experience-builder-page-not-default-jh

Updated the Diagnostics topic with steps to configure Login & Registration defaults  back to Experience Builder Page.

JennyHajee committed:
Update diagnostics.adoc

ferferrarazzo commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
+. From the search results, select the community name.
+. The *Login and Registration* page shows the community account in the *Accounts* field.
+. From the top, right of the community page, click the refresh icon. The *Experience Builder Page* is set for *Login Page Type*, *Forgot Password*, and *Registration Page Type* fields.
```
```suggestion
. From the top, right of the community page, click the refresh icon. The *Experience Builder Page* is set for *Login Page Type* and *Registration Page Type* fields.
```

ferferrarazzo commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
+
+. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign*.
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
+. From the search results, select the community name.
```
```suggestion
. From the search results, select the community name 
```
+ account (for example: 'CommunityName Account'.)

JennyHajee committed:
Update modules/ROOT/pages/diagnostics.adoc

added period.

Co-authored-by: ferferrarazzo <fernando.ferrarazzo@gmail.com>

JennyHajee committed:
Update modules/ROOT/pages/diagnostics.adoc

removed Forgot Password

Co-authored-by: ferferrarazzo <fernando.ferrarazzo@gmail.com>

JennyHajee committed:
Update diagnostics.adoc

Updated to:
3. From the search results, select the community name. The account name is the community name + account, for example, MyCommunityName Account.

kbostrom commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
-. Click *Save*.
-. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign* to verify that the account is assigned correctly.
-. Fix the remaining items in the list.
+If a community was created but not published, when diagnostics *Fix All* is run for the member user, the *Default Page* for the *Login and Registration* pages will display and not the *Experience Builder Page*.
```
```suggestion
If a community is created but not published, when diagnostics *Fix All* is run for the member user, the *Default Page* for the *Login and Registration* pages displays instead of the *Experience Builder Page*.
```

kbostrom commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
+If a community was created but not published, when diagnostics *Fix All* is run for the member user, the *Default Page* for the *Login and Registration* pages will display and not the *Experience Builder Page*.
+
+After the items from *Community profile* through *Account created* are configured
+, fix the diagnostics for the *Account assigned*:
```
```suggestion
After the items from *Community profile* through *Account created* are configured, fix the diagnostics for the *Account assigned*:
```

kbostrom commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
+, fix the diagnostics for the *Account assigned*:
+
+. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign*.
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
```
```suggestion
. From *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
```

kbostrom commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
+
+. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign*.
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
+. From the search results, select the community name. The account name is the community name + account, for example, MyCommunityName Account.
```
If the user is selecting the community name, do we really need to define the account name?
Maybe reword this if so, to something like
```suggestion
. From the search results, select the community name from the list of account names, which are displayed in the format MyCommunityName Account.
```

kbostrom commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
+. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign*.
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
+. From the search results, select the community name. The account name is the community name + account, for example, MyCommunityName Account.
+. The *Login and Registration* page shows the community account in the *Accounts* field.
```
This is not a step, because there is no user action. It should be a plain paragraph at the same level as the previous step:

. From the search results, select the community name from the list of account names.
+
The *Login and Registration* page shows the community account in the *Accounts* field.

kbostrom commented on modules/ROOT/pages/diagnostics.adoc:
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
+. From the search results, select the community name. The account name is the community name + account, for example, MyCommunityName Account.
+. The *Login and Registration* page shows the community account in the *Accounts* field.
+. From the top, right of the community page, click the refresh icon. The *Experience Builder Page* is set for *Login Page Type* and *Registration Page Type* fields.
```
```suggestion
. From the top right of the community page, click the refresh icon.
+
The *Experience Builder Page* is set for *Login Page Type* and *Registration Page Type* fields.
```

kbostrom reviewed (dismissed):
PR; should have been ER

JennyHajee committed:
Update modules/ROOT/pages/diagnostics.adoc

accepted changes

Co-authored-by: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

JennyHajee committed:
Update modules/ROOT/pages/diagnostics.adoc

accepted changes

Co-authored-by: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

JennyHajee committed:
Update modules/ROOT/pages/diagnostics.adoc

accepted changes

Co-authored-by: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

JennyHajee committed:
Update modules/ROOT/pages/diagnostics.adoc

accepted changes

Co-authored-by: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

JennyHajee committed:
Update modules/ROOT/pages/diagnostics.adoc

accepted changes

Co-authored-by: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

JennyHajee committed:
Merge branch 'latest' into docs-13092-acm-experience-builder-pages-not-default-jh

kbostrom reviewed (approved):
PR
